rand_chacha/src/guts.rs

```diff
@@ -0,0 +1,245 @@
+// Copyright 2018 Developers of the Rand project.
+// Copyright 2019 The CryptoCorrosion Contributors
```

Comment:
Should `// Copyright 2020 Developers of the Rand project.` be added?

Reply:
Wrong year you mean? I guess, but don't think it matters much.

Reply:
I assumed the 2018 meant that CryptoCorrosion copied it from Rand. :)

Reply:
I think the only sensible way to interpret these things is *the mentioned party may have contributed to this*, be that recently or not. Git history is much more reliable. From what I understand (as a lay person), it doesn't have much legal meaning either.